### ISSUE TITLE
Sales, Transactions, Card/Account Holder Data

### ISSUE THREAD

**Issue by johnbeallgsa** [GSA/smartpay-website#768]
Replace the link with FY 2024

[Sales, Transactions, and Card/Account Holder Data (FY 2009–February 2024) [XLSX, 7.3MB]](https://smartpay.gsa.gov/files/stats-tool.xlsx) – Spreadsheets containing current statistics for GSA SmartPay Purchase, Travel, and Fleet business lines

**Comment by JennaySDavis** [comment]
Use this file until the complete FY24 file is available [StatsTool_FY25_M2_v1_EXTERNAL.xlsx](https://github.com/user-attachments/files/18300396/StatsTool_FY25_M2_v1_EXTERNAL.xlsx)

**Comment by johnbeallgsa** [comment]
Yes, with the name of Sales, Transactions, and Card/Account Holder Data (FY 2009– August 2024) 

Can we also rename the file name to StatsTool_FY24AUG_M2_v1_External.xlsx or something similar

**Comment by JennaySDavis** [comment]
**768 Acceptance Criteria**



| Pass/Fail | Description |
| --- | --- | 
| Pass | File Name Updated, File Updated|



**Comments/Additional Notes**
N/A


**ADA Compliance (Automated scan via Chrome Lighthouse)**

| Criteria | Score |
| --- | --- | 
| Performance | 98 |
| Accessibility | 100 |
| Best Practices | 100 |


Passed 01/03/2025 - JSD

**Comment by johnbeallgsa** [comment]
Looks great. Moving to Done.
